atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -155,6 +178,73 @@ public void userAgentsPresentOnRequestsWithLeaderBlockConfigured() throws IOExce
     }
 
     @Test
+    public void remoteCallsStillMadeIfPingableLeader404s() throws IOException, InterruptedException {
+        availableServer.stubFor(LEADER_UUID_MAPPING.willReturn(aResponse().withStatus(404)));
+        when(config.leader()).thenReturn(Optional.of(ImmutableLeaderConfig.builder()
```

Comment:
can we add a test that we use the remote if there is a connection error (not sure if we can stub that here)? also might make it easier if we refactor the common bits of these two methods